frontend/packages/kubevirt-plugin/integration-tests-cypress/tests/template/create-template-from-wizard.spec.ts

```diff
@@ -59,7 +59,7 @@ const pxeTemplate: VirtualMachineData = {
   networkInterfaces: [nic0],
 };
 
-describe('Test VM creation', () => {
+describe('Test Template creation', () => {
```

Comment:
nitpick: `Template` shouldn't be capitalized here.

Reply:
NP